doc/locale/ja/LC_MESSAGES/reference.po

```diff
@@ -28954,6 +28954,200 @@ msgstr ""
 msgid "Outputs reading of token."
 msgstr "トークンの読みを出力します。"
 
+msgid "``TokenNgram``"
+msgstr ""
+
+msgid ""
+"``TokenNgram`` can change define its behavior dynamically via its options. "
+"For example, we can use it as unigram, bigram, trigram on with changing "
+"``n`` option value as below."
+msgstr ""
+"``TokenNgram`` はオプションによって、動的に動作を定義できます。例えば、以下の"
+"ように ``n`` オプションの値を変更する事によって、ユニグラム、バイグラム、トリ"
+"グラムとして使えます。"
+
+msgid "Uni-gram:"
+msgstr "ユニグラム:"
+
+msgid "Bi-gram:"
+msgstr "バイグラム:"
+
+msgid "Tri-gram:"
+msgstr "トリグラム:"
+
+msgid "This tokenizer also has options other than the above."
+msgstr "このトークナイザーには、上記以外のオプションもあります。"
+
+msgid "``TokenNgram`` has optional parameter."
+msgstr "``TokenNgram`` は、省略可能な引数があります。"
+
+msgid ""
+"If we don't use options, ``TokenNgram`` is the same behavior for "
+"``TokenBigram``."
+msgstr ""
+"オプションを使わない場合、 ``TokenNgram`` は ``TokenBigram`` と同じ動作になり"
+"ます。"
+
+msgid ""
+"``TokenNgram`` We can also specify multiple options at the same time except "
+"the above examples."
+msgstr "上記の例以外にも複数のオプションを組み合わせて指定できます。"
+
```

Comment:
Missing ``TokenNgram`` from translation. Is this expected?

Reply:
`TokenNgram` is needless.
I've removed it.